.github/workflows/build.yaml

```diff
@@ -47,4 +47,4 @@ jobs:
         run: cd build && make -j $(nproc)
       - name: CTest with single thread
         timeout-minutes: 40
-        run: cd build && ASAN_SYMBOLIZER_PATH=/usr/bin/llvm-symbolizer-8 ASAN_OPTIONS=fast_unwind_on_malloc=0 ctest --timeout 300 --output-on-failure
```

Comment:
`ASAN_SYMBOLIZER_PATH=/usr/bin/llvm-symbolizer-8`, the variable useless ?

Reply:
refer to https://stackoverflow.com/questions/38079761/why-does-asan-symbolizer-path-no-longer-work-with-version-adorned-binaries